Java/Primitive Data Type/Casting/Type_Casting.ipynb

```diff
@@ -0,0 +1,297 @@
+{
+  "nbformat": 4,
+  "nbformat_minor": 0,
+  "metadata": {
+    "colab": {
+      "name": "Untitled1.ipynb",
+      "provenance": []
+    },
+    "kernelspec": {
+      "name": "python3",
+      "display_name": "Python 3"
+    },
+    "language_info": {
+      "name": "python"
+    }
+  },
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "C5tzpFj4S60F"
+      },
+      "source": [
+        "# **CASTING**\n",
+        "Type casting is the process to convert a data type into another data type.\n"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "Hc-dRqf_Vf4J"
+      },
+      "source": [
+        "int a=7;"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "P7G7MtOqVjD1"
+      },
+      "source": [
+        "In the above example, we are assigning 0 (an integer value) to 'a' (an integer variable) and that works fine.\n",
```

Comment:
In the example the contributor has int a=`7`, but in this line, the contributor has written "we are assigning  `0`"